### PR TITLE
Translation helper updates in notes/_card.html.erb - one t function remains

### PR DESCRIPTION
**Pull request by stephaniequintana**
I was unable to complete the following edits:
- 49 
This item does not exist in the code.
- 57
The t function is still present in the code. I did not update it due to a difference in the conditional statement, namely:

code: <% if @current_user && @current_user.can_moderate? %>
file: <% if @current_user && ['admin','moderator'].include?(@current_user.role) %>

This is the only place in the file where a t function remains.
Please let me know how you would like me to proceed.

Fixes #0000 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by TildaDares** [comment]
Hi @stephaniequintana, can you add thee issue number in the format `Fixes #0000`? The issue number is the number at the top of the issue. Thanks!

**Comment by stephaniequintana** [comment]
Fixes #8508

**Comment by stephaniequintana** [comment]
@TildaDares - Is there a more effective way to edit the PR or is the comment 'Fixes #8508' the best way to make this correction?

**Comment by TildaDares** [comment]
`Fixes #0000` is used to connect the issue to this PR so that when this PR is merged, the issue is closed at the same time.

**Comment by stephaniequintana** [comment]
@TildaDares - Thank you. As far as making specific edits to a PR after it has been created, is there a best practice or means of doing so or is this comment thread the only way/place to update the information?

**Comment by TildaDares** [comment]
You can choose to use the web interface to make edits to a PR. If you have the local environment installed, you can push changes from there too. 

Do you have the local environment setup?

**Comment by stephaniequintana** [comment]
@TildaDares - I apologize for the delayed response...I'm working on getting my local environment setup. I'm running into multiple roadblocks with zsh on my Mac - this is apparently due to the Apple system already having it's own version of Ruby (?). Seemingly no matter what I do, I keep getting "zsh: command not found" errors when I input anything rvm related...I've found some great resources to help remedy the issue. I'm currently downloading the Ruby Version Manager and setting up paths and hopefully will be set up soon. 

With this being a Mac issue, perhaps someone here has run into this already? I would definitely appreciate any existing solutions at hand. If I am unable to finish the setup on my own, I will reach out to the team for help. Although, I prefer to have my local environment set up, I do see, [here](https://github.com/publiclab/plots2/issues/3840), that  Codenvy is listed as an alternative. Would using this route still allow me to contribute?

I just wanted to give you a heads up as to where I am and let you know that I will document the issues I am having along with the solution I find. Hope you're having a great night!

**Comment by TildaDares** [comment]
@stephaniequintana I’m not sure about codeenvy but I know we use gitpod https://gitpod.io/#https://github.com/publiclab/plots2 for development. [Gitpod](https://www.gitpod.io/) makes development very fast.

If you need our help, feel free to reach out to us.

**Comment by stephaniequintana** [comment]
@TildaDares Thank you! I'm making headway. I finally got Ruby downloaded and am currently trying to run `rake db:setup` but am getting `Gem::LoadError: mysql2 is not part of the bundle. Add it to your Gemfile` even though it is in the Gemfile. It may be a version issue...I've found a few resource so hopefully those work out b/c I'd much prefer to get my local environment setup. Thanks so much for reaching out.

File: app/views/notes/_card.html.erb [hunk 1]
```diff
@@ -52,16 +52,16 @@
       <a class="ellipsis bottom-right" data-toggle="dropdown"><i class="fa fa-ellipsis-h" style="color : #666; font-size:15px; float:right; cursor:pointer;"></i></a>
 
       <ul class="dropdown-menu" style = "width: 150px; font-size:10px;">
-        <li><a>Total Views: </i> <%= number_with_delimiter(node.views) %> <span class="d-none d-lg-inline"><%= t('notes._notes.views') %></span></a></li>
+      <li><a>Total Views: </i> <%= number_with_delimiter(node.views) %> <span class="d-none d-lg-inline"><%= translation('notes._notes.views') %></span></a></li>
         <li><a>Total Likes: <%= node.cached_likes %></a></li>
         <div class="content" style="width: 100%" >
           <% if @compact.nil? %>
 
             <li class="buttons mb-2"><% if @current_user && @current_user.can_moderate? %><a style="width: 90px" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %></li>
```
Comment (jywarren) [inline app/views/notes/_card.html.erb:60]: Hi @stephaniequintana is this the line where you weren't sure? It should be OK to change the `t()` function here. The code is very similar, and has the same function, and shouldn't interfere with your changes. 

If you like you can accept this suggestion:

```suggestion
            <li class="buttons mb-2"><% if @current_user && @current_user.can_moderate? %><a style="width: 90px" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= translate('notes._notes.spam') %></a><% end %></li>
```

Thank you!!

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Hi @stephaniequintana -- i'm going to try to help with your mysql gem issues in #4936 -- but unfortunately I think i have some bad news for this PR, which is not at all your fault. 

Basically this template file, along with the files in a related PR (https://github.com/publiclab/plots2/pull/10420), is a little different than most, because sometimes we use it in an unusual way within a Rails model file, in `/app/models/concerns/node_shared.rb`. The `translate()` helper which we normally use is not accessible when run in this way. 

I got really stuck trying to fix this in https://github.com/publiclab/plots2/pull/10420 and unfortunately it affects this PR as well, which is why the unit tests failed:

```
Progress: |=============================================================ERROR NodeSharedTest#test_that_NodeShared_can_be_used_to_convert_short_codes_like_[nodes:grid:foo]_into_tables_which_list_nodes_with_image_thumbnails (21.64s)
Minitest::UnexpectedError:         RuntimeError: Wrapped undumpable exception for: ActionView::Template::Error: undefined method `translation' for #<#<Class:0x000055ae9[89](https://github.com/publiclab/plots2/runs/5618369530?check_suite_focus=true#step:5:89)e20e0>:0x000055ae974c[90](https://github.com/publiclab/plots2/runs/5618369530?check_suite_focus=true#step:5:90)78>
        Did you mean?  translate
            app/views/notes/_card.html.erb:55:in `_app_views_notes__card_html_erb__994381742518529536_101940'
            app/views/notes/_notes.html.erb:9:in `block in _app_views_notes__notes_html_erb__160113718156310036_101[92](https://github.com/publiclab/plots2/runs/5618369530?check_suite_focus=true#step:5:92)0'
            app/views/notes/_notes.html.erb:6:in `each'
            app/views/notes/_notes.html.erb:6:in `each_with_index'
            app/views/notes/_notes.html.erb:6:in `_app_views_notes__notes_html_erb__160113718156310036_101920'
            app/views/grids/_thumbnail.html.erb:2:in `_app_views_grids__thumbnail_html_erb___1238360[93](https://github.com/publiclab/plots2/runs/5618369530?check_suite_focus=true#step:5:93)56924[97](https://github.com/publiclab/plots2/runs/5618369530?check_suite_focus=true#step:5:97)506_[101](https://github.com/publiclab/plots2/runs/5618369530?check_suite_focus=true#step:5:101)900'
            app/models/concerns/node_shared.rb:345:in `data_string'
            app/models/concerns/node_shared.rb:53:in `block in nodes_thumbnail_grid'
            app/models/concerns/node_shared.rb:39:in `gsub'
            app/models/concerns/node_shared.rb:39:in `nodes_thumbnail_grid'
            test/unit/node_shared_test.rb:31:in `block in <class:NodeSharedTest>'
```

I'm so sorry but unfortunately I'm not sure how to address this. I can try a few more things but there's a strong possibility this may be stuck until we figure out how to handle this situation. I'm so sorry as i know it's not a great first-time experience. Could we help you find another first-timers-only issue to try?

File: app/views/notes/_card.html.erb (hunk 1 above)
Comment (stephaniequintana) [reply app/views/notes/_card.html.erb:60]: Yes, that was the function in question. Thank you!

**Comment by stephaniequintana** [comment]
@jywarren This was a fantastic first time issue - I think I would have learned much less had it have gone smoothly 😄 ...Yes, I'd definitely like to do another first-timers-only issue before I begin helping others and attempting some of the more complex ones. If you have one for me I'd be happy to get started else I'll continue to search in the meantime.

**Comment by jywarren** [comment]
Hi @stephaniequintana would you like to try this one? https://github.com/publiclab/plots2/issues/10820

**Pull request closed without merging**